code/modules/research/techweb/all_nodes.dm

```diff
@@ -1220,6 +1221,10 @@
 		"light_replacer",
 		"paint_remover",
 		"spraybottle",
```

Comment:
extra space reeee
```suggestion
		"spraybottle",
		//SKYRAT EDIT START - RESEARCH DESIGNS
```

Reply:
the space is how its done in many other instances within this file.